src/components/users/upcomingShows/UpcomingList.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import {
+  List,
+  ListItem,
+  ListItemText,
+  ThemeProvider,
+} from "@material-ui/core";
+import moment from "moment";
+import Modal, { ModalProvider } from "styled-react-modal";
+import { UpcomingDetail } from "../../index";
+
+function UpcomingList(props) {
+  const { concerts } = props;
+  const [showOpen, updateShowOpen] = useState(false);
+
+  // helper function to toggle overlay
+  const toggleShowOpen = () => {
+    updateShowOpen(!showOpen);
+  };
+
+  const handleClick = () => {
+    toggleShowOpen();
+  };
+
+  return (
+    <List>
+      {concerts.map((concert, i) => {
+        return (
+          <div key={i}>
+            <ListItem onClick={handleClick}>
+              <ListItemText
+                primary={concert.bandname}
+                secondary={`${moment(concert.starttime).format(
+                  "yyyy-MM-DD"
+                )} starts ${moment(concert.starttime).format("hh:mm A")} on ${
+                  concert.stage.name
+                }`}
+              />
+            </ListItem>
+            {/* render show concert details as overlay */}
+            <ThemeProvider theme={{}}>
+              <ModalProvider>
+                <StyledModal
+                  isOpen={showOpen}
+                  onBackgroundClick={toggleShowOpen}
+                  onEscapeKeydown={toggleShowOpen}
+                >
+                   <UpcomingDetail concert={concert}/>
```

Comment:
@AnnaTielke Nicht das du dich wunderst. Ich konnte hier nicht die ConcertDetails.js component verwenden. Irgendwie mag materialize die Idee nicht die component in der map zu rendern. alles andere war zu aufwendig. daher gibts (erstmal) eine weitere simple detail component ohne materialize hier.